build/package/.mongocli.goreleaser.yml

```diff
@@ -86,8 +84,23 @@ changelog:
     - '^test:'
     - '^task:'
     - '^build(deps):'
+publishers:
+  - name: sign-linux-artifacts-garasign-gpg
+    ids:  # IDs of the artifacts we want to sign
+      - linux_archives
+      - linux_package
+    cmd: "./build/package/linux_notarize.sh"
+    env:
+      - VERSION={{ .Version }}
+      - TOOL_NAME=mongocli
+      - ARTIFACTORY_USERNAME={{.Env.ARTIFACTORY_USERNAME}}
+      - ARTIFACTORY_PASSWORD={{.Env.ARTIFACTORY_PASSWORD}}
+      - GRS_USERNAME={{.Env.GRS_USERNAME}}
+      - GRS_PASSWORD={{.Env.GRS_PASSWORD}}
+      - LINUX_FILE={{ abs .ArtifactPath }}
 release:
   prerelease: auto
   name_template: "MongoDB CLI {{.Version}}"
   extra_files:
     - glob: ./dist/*.msi
+    - glob: ./dist/*.asc
```

Comment:
we don't need to publish these as they'll always be available in https://pgp.mongodb.com/

Reply:
My understanding is that users will have to:

- Download the artifact (example AtlasCLI-111.rpm)
- Download the related .sign file (example AtlasCLI-111.rpm.sign)
- Download the AtlasCLI PGP public key
- Import the public key
```bash
gpg --import server-7.0.5.asc
```
- Verify the signature of the artifact

```
gpg --verify mongodb-atlas-cli_22.0.27_linux_arm64.deb.asc mongodb-atlas-cli_22.0.27_linux_arm64.deb
gpg: Signature made Mon 29 Jan 10:21:01 2024 GMT
gpg:                using RSA key BBDCC63DEAA88AF8E2B8870835565AA4C7ECBC6A
gpg: Good signature from "Atlas CLI Release Signing Key <packaging@mongodb.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: BBDC C63D EAA8 8AF8 E2B8  8708 3556 5AA4 C7EC BC6A
```

The MongoDB binary has this process documented: https://www.mongodb.com/docs/manual/tutorial/verify-mongodb-packages/#verify-linux-macos-packages

Let me know if I am missing something here

Reply:
I guess for server is `https://pgp.mongodb.com/server-<version>.asc` for ops manager is `https://pgp.mongodb.com/opsmanager-<version>.asc`.

I guess for atlascli would be something like `https://pgp.mongodb.com/atlascli.asc` and mongocli `https://pgp.mongodb.com/mongocli.asc`.

The gist is not on github because our friends from devprod need to generate this file prior to releasing it

Reply:
we now have https://pgp.mongodb.com/atlas-cli.asc and https://pgp.mongodb.com/mongodb-cli.asc

Reply:
Which means we don't need to send these files to GitHub release or s3, just document where they are.